### PR TITLE
Clean up metadata of readmes

### PR DESCRIPTION
**Pull request by janhohenheim**
- Move stuff into a `docs` directory
- Change some names

I'll change the actual content of the files in a follow-up

### CODE REVIEW

**Comment by benfrankel** [comment]
Eh I think uppercase names for README and LICENSE are more standard. The moves to `docs/` look good though.

**Comment by janhohenheim** [comment]
@benfrankel more standard, yes, but also more annoying.

**Comment by benfrankel** [comment]
I'd opt for standard in this case. Every repo has license and readme files, and they're always uppercase.

**Comment by janhohenheim** [comment]
@benfrankel I've renamed the files.

File: README.md [hunk 1]
```diff
@@ -23,9 +23,14 @@ Navigate to the top of [this GitHub repository](https://github.com/TheBevyFlock/
 ![example](TODO)
 
 Next, clone your new Github repository to a local repository and create an `Initial commit` with the following changes:
-- Delete `LICENSE` and `README` files.
+
+- Delete the following.
+  - `README.md`
+  - All `LICENSE-*.txt` files
+  - the `docs` directory
```
Comment (benfrankel) [inline README.md:30]:
```suggestion
- Delete `LICENSE`, `README`, and `docs/` files.
```
I think it looks less scary this way than nested list.

**Review by benfrankel — CHANGES_REQUESTED** [review]
I think header of `docs/workflows.md` needs to be changed from `CI / CD` as well.

**Comment by janhohenheim** [comment]
> I think header of `docs/workflows.md` needs to be changed from `CI / CD` as well.

I intentionally left that alone since I'm changing the contents next.

**Review by benfrankel — APPROVED** [review]

**Pull request merged**